### PR TITLE
Added Image Compression 

### LINKED ISSUE #602: Long s3 media load times
Retrieving s3 files often takes several seconds or more

### LINKED ISSUE #603: Flutter app performance
App performance and speed seems a bit laggy. For example, clicking on notifications and navigating back has a slight delay. Viewing a photo and going back also crashes the app occasionally.

### PR DESCRIPTION
**Pull request by Nash0x7E2**
This PR adds support image compression. To achieve this, the package `FlutterNativeImage` was added. 

- Average compression time: ~200 ms 
- Compression percentage: 70

 - Closes #634 
 - Closes  #603 
 - Closes  #602

### CODE REVIEW

**Comment by orestesgaolin** [comment]
Imho 70% is a bit too much. What about 85/90?

File: pubspec.yaml [hunk 1]
```diff
@@ -40,15 +40,20 @@ dependencies:
   device_preview: 0.4.1+1
   intl: ^0.16.1
   audioplayers: 0.15.1
+  flutter_native_image: 0.0.5
   flutter_audio_recorder: ^0.5.5
-  hive: 1.4.1+1
-  hive_flutter: 0.3.0+1
+  hive:
+  hive_flutter:
```
Comment (orestesgaolin) [inline pubspec.yaml:46]: Why do we need the fork?

Comment (Nash0x7E2) [reply pubspec.yaml:46]: Conflict with Flutter intl on dev

Comment (orestesgaolin) [reply pubspec.yaml]: Let's fix version to 1.17/stable and this issue is resolved 
image

**Pull request merged**